payload/import/mkwRegistry.hpp

```diff
@@ -48,6 +128,40 @@ enum class Course {
     N64Skyscraper = 0x29,
 };
 
+static constexpr bool IsCharacterValid(Character character)
+{
+    switch (character) {
+    case Character::Mario... Character::SmallMiiOutfitBFemale:
+    case Character::MediumMiiOutfitAMale... Character::MediumMiiOutfitBFemale:
+    case Character::LargeMiiOutfitAMale... Character::LargeMiiOutfitBFemale:
+        return true;
+    default:
+        return false;
+    }
+}
+
+static constexpr bool IsVehicleValid(Vehicle vehicle)
+{
+    return vehicle >= Vehicle::StandardKartSmall && vehicle <= Vehicle::Phantom;
+}
```

Comment:
Check the gamemode in `IsCombinationValid` to determine which function to use for validation.

```suggestion
static constexpr bool IsVehicleValidVS(Vehicle vehicle)
{
    return vehicle >= Vehicle::StandardKartSmall && vehicle <= Vehicle::Phantom;
}

static constexpr bool IsVehicleValidBT(Vehicle vehicle)
{
    switch (vehicle) {
    case Vehicle::StandardKartSmall:
    case Vehicle::StandardKartMedium:
    case Vehicle::StandardKartLarge:
    case Vehicle::StandardBikeSmall:
    case Vehicle::StandardBikeMedium:
    case Vehicle::StandardBikeLarge:
        return true;
    default:
        return false;
    }
}
```

Reply:
Implemented in commit [21bbff1](https://github.com/WiiLink24/wfc-patcher-wii/commit/21bbff10dc858823f734faf80d808bed8ce3eb23).